feature/record/src/main/java/com/ddd/carssok/feature/record/accident/RecordAccidentScreen.kt

```diff
@@ -14,33 +30,81 @@ import androidx.compose.material3.FabPosition
 import androidx.compose.material3.Icon
 import androidx.compose.material3.Scaffold
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.res.colorResource
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
 import com.ddd.carssok.core.designsystem.TypoStyle
 import com.ddd.carssok.core.designsystem.component.Appbar
 import com.ddd.carssok.core.designsystem.component.CarssokButton
+import com.ddd.carssok.core.designsystem.component.Progress
 import com.ddd.carssok.core.designsystem.component.TypoText
 import com.ddd.carssok.core.designsystem.component.input.InputImageBox
 import com.ddd.carssok.core.designsystem.component.input.InputTextBox
+import com.ddd.carssok.core.util.FileUtils.optimizeBitmap
 import com.ddd.carssok.feature.record.R
+import kotlinx.coroutines.flow.collectLatest
+import java.io.File
 
 @Composable
-fun RecordAccidentRoute(onClickedBack: () -> Unit) {
+fun RecordAccidentRoute(
+    viewModel: RecordAccidentViewModel = hiltViewModel(),
+    onClickedBack: () -> Unit
+) {
+    val uiState by viewModel.uiState.collectAsState()
+
+    LaunchedEffect(Unit) {
+        viewModel.event.collectLatest {
+            when (it) {
+                is RecordAccidentViewModel.Event.OnSuccess -> onClickedBack.invoke()
+            }
+        }
+    }
     RecordAccidentScreen(
+        uiState,
         onClickedBack = onClickedBack,
-        onClickedDataPicker = {}
+        onUpdateDate = viewModel::updateDate,
+        onChangedMemo = viewModel::updateMemo,
+        onChangedLocationMemo = viewModel::updateLocationMemo,
+        onPhotoChanged = viewModel::addPicture,
+        onClickedRemovePhoto = viewModel::removePicture,
+        onClickedSave = viewModel::saveRecordAccident
     )
 }
 
+@OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun RecordAccidentScreen(
+    uiState: RecordAccidentViewModel.UiState,
     onClickedBack: () -> Unit,
-    onClickedDataPicker: () -> Unit
+    onUpdateDate: (String) -> Unit,
+    onChangedMemo: (String) -> Unit,
+    onChangedLocationMemo: (String) -> Unit,
+    onPhotoChanged: (Uri) -> Unit,
+    onClickedRemovePhoto: (Int) -> Unit,
+    onClickedSave: (List<File>) -> Unit
 ) {
+    val datePicker = DatePickerDialog(
+        LocalContext.current,
+        { _: DatePicker, selectedYear: Int, selectedMonth: Int, selectedDayOfMonth: Int ->
+            onUpdateDate(
+                "$selectedYear-${"%02d".format(selectedMonth + 1)}-$selectedDayOfMonth"
```

Comment:
```suggestion
                "$selectedYear-${"%02d".format(selectedMonth + 1)}-${"%02d".format(selectedDayOfMonth)}"
```
제가 이부분 버그나서 보니까, month, date 모두 %02d 로 넘겨야 되더라구용, 안그러면 계산이 달라져요